docs/source/developers/contributing/prerequisites.rst

```diff
@@ -3,68 +3,64 @@
 Prerequisites
 =============
 
-Prerequisites for working with the repo
------------------------------------------
-This section lists the prerequisites for working with the repository. Most contributors should start with the basic prerequisites. 
+This section lists the prerequisites for working with the repository.
+Most contributors should start with the basic prerequisites. 
 Depending on the task you need to perform, you may need to install more tools.
 
-We also provide a `Devcontainer <https://code.visualstudio.com/docs/devcontainers/containers>`_ for working with this repository without installing prerequisites. 
-Keep reading for instructions.
+We highly recommend using VS Code for development.
+We provide a `dev container <https://code.visualstudio.com/docs/devcontainers/containers>`_ that will automatically install all of the prerequisites for you.
+It will take a while to download and install the first time you run it, but subsequent runs will be much faster.
 
-Operating system
-^^^^^^^^^^^^^^^^^^^^
-We support developing on macOS, Linux and Windows with `WSL <https://docs.microsoft.com/windows/wsl/install>`_.
+VS Code Setup
+-------------
 
-Asking for help
-^^^^^^^^^^^^^^^^^^^^
-If you get stuck installing any of our dependencies, you can raise an issue or ask for help in our `discord <https://discordapp.com/channels/1116864463832891502/11168644638328915074>`_.
+We recommend using VS Code for Paraglider development because we provide a `dev container <https://code.visualstudio.com/docs/devcontainers/containers>`_ that will automatically install all of the prerequisites for you.
 
-Required tools
-^^^^^^^^^^^^^^^^^^^^
-This is the list of core dependencies to install for the most common tasks. In general we expect all contributors to have all of these tools present:
+#. `Get VS Code set up to use dev containers <https://code.visualstudio.com/docs/devcontainers/containers#_getting-started>`_.
+#. `In VS Code, open the cloned repository in a container <https://code.visualstudio.com/docs/devcontainers/containers#_quick-start-open-an-existing-folder-in-a-container>`_.
 
-- `Git <https://git-scm.com>`_
-- `Go <https://golang.org/>`_
-- `golangci-lint <https://golangci-lint.run>`_
-- `protoc <https://grpc.io/docs/protoc-installation>`_
-- make
-
-  * Linux: Install the ``build-essential`` package:
-
-    .. code-block:: console
-    
-        $ apt install build-essential
+   .. note::
   
-  * Mac:
+        The first time you open the repository in a container, it will take a while to download and install the prerequisites.
```

Comment:
This is redundant with the above, so maybe remove lines 11 and 12? Or consolidate in some other way.